packages/clearpay-integration/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "references": [
+      {
+          "path": "./tsconfig.lib.json"
+      },
```

Comment:
You can remove these lines , since tsconfig.lib.json file is not required for integration package